core/src/main/java/com/amplifyframework/core/model/annotations/AuthRule.java

```diff
@@ -40,28 +40,43 @@
 @Target(ElementType.ANNOTATION_TYPE)
 public @interface AuthRule {
     /**
-     * Defines the strategy for this rule.  This is the only required field.
+     * Defines the strategy for this rule. This is the only required field.
      *
      * @return AuthStrategy for this {@link AuthRule}
      */
     AuthStrategy allow();
 
     /**
-     * Used for owner authorization.  Defaults to "owner" when using AuthStrategy.OWNER.
+     * Used to specify the type of authorization being used.
+     *
+     * @return mode of authorization
+     */
+    String provider() default "";
+
+    /**
+     * Used for owner authorization.
      *
      * @return name of a {@link ModelField} of type String which specifies the user which should have access
      */
     String ownerField() default "";
 
     /**
-     * Used to specify a custom claim.  Defaults to "username" when using AuthStrategy.OWNER.
+     * Used to specify a custom claim.
```

Comment:
Did you mean to remove "Defaults to "username" when using AuthStrategy.OWNER."?  I think that comment is still accurate.

Reply:
I think this statement is inaccurate inside `@AuthRule` annotation since default is "". We only decide the default value once it gets passed over to `AuthRule` object.